app/components/search/SearchResults/SearchResults.module.scss

```
@@ -32,6 +32,15 @@
   }
 }
 
+.noResultsFound {
+  display: none;
+  &.showNoResultsMsg {
+    display: block;
+    margin: 35px auto 65px;
+    text-align: center;
+  }
+}
```

Since these are meant to be attached to the same element, it might be easier to reason about if you separate the displayedness of the component from the other properties of element, even if those other properties are not visible when the element is hidden. What if you named and organized the CSS rules like this?

```suggestion
.noResultsMessage {
  margin: 35px auto 65px;
  text-align: center;

  &.hidden {
    display: none;
  }
}
```

and updated the JSX code's logic accordingly, such that the `hidden` class gets applied by default, and is only removed when there are no results?

---

Much better, thanks